src/Development/IDE/Plugin/Completions.hs

```diff
@@ -71,15 +71,27 @@ produceCompletions = do
         case (ms, sess) of
             (Just (ms,imps), Just sess) -> do
               let env = hscEnv sess
-              res <- liftIO $ tcRnImportDecls env imps
+              -- We do this to be able to provide completions of items that are not restricted to the explicit list
+              let imps' = map dropListFromImportDecl imps
+              res <- liftIO $ tcRnImportDecls env imps'
               case res of
                   (_, Just rdrEnv) -> do
-                      cdata <- liftIO $ cacheDataProducer env (ms_mod ms) rdrEnv imps parsedDeps
+                      cdata <- liftIO $ cacheDataProducer env (ms_mod ms) rdrEnv imps' parsedDeps
                       return ([], Just cdata)
                   (_diag, _) ->
                       return ([], Nothing)
             _ -> return ([], Nothing)
 
+-- Drop any explicit imports in ImportDecl if not hidden
+dropListFromImportDecl :: GenLocated SrcSpan (ImportDecl GhcPs) -> GenLocated SrcSpan (ImportDecl GhcPs)
```

Comment:
This change is already merged, so I don't understand why it shows up in your PR. 

Could you please rebase on top of master and force push? I will hold my review until then.